_posts/2018-02-20-announcing-riff-0-0-4.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: single
+title: "Announcing riff 0.0.4"
+header:
+  overlay_image: /images/winter-night.jpg
+excerpt:
+  With a new go CLI, gRPC, and improved node invoker.
+permalink: /blog/announcing-riff-0-0-4/
+---
+
+We are happy to announce another new release of riff. Thank you, once again, everyone
```

Comment:
"We are happy to announce a new release of riff" or "We are happy to announce another release of riff" sounds better to me